Services/Excel/classes/class.ilExcel.php

```diff
@@ -542,7 +543,8 @@ public function setBorders($a_coords, $a_top, $a_right = false, $a_bottom = fals
 	 */
 	function getCoordByColumnAndRow($pColumn = 0, $pRow = 1)
 	{
-		$columnLetter = Coordinate::stringFromColumnIndex($pColumn);
+		$pColumn = ++$pColumn;
+		$columnLetter = Coordinate::stringFromColumnIndex($pColumn + 1);
```

Comment:
@bheyser / @xus : `return Coordinate::stringFromColumnIndex($pColumn + 2);` would be much easier to understand :-)

Reply:
Shit (!) I will provide another PR :-D Thx for the hint (!) :-D